src/bpftrace.h

```diff
@@ -202,12 +213,8 @@ class BPFtrace
   bool demangle_cpp_symbols_ = true;
   bool resolve_user_symbols_ = true;
   bool debug_output_ = false;
-  enum class UserSymbolCacheType
-  {
-    per_pid,
-    per_program,
-    none,
-  } user_symbol_cache_type_;
+  bool verify_llvm_ir_ = false;
+  UserSymbolCacheType user_symbol_cache_type_;
```

Comment:
It feels like introduction of these two fields could be moved into separate commit(s).